package.json

```diff
@@ -9,10 +9,10 @@
   "author": "@lennymarkus",
   "license": "Apache-2.0",
   "dependencies": {
+    "express": "^5.1.0",
```

Comment:
This part I'm not ok with. 
I see you wanted to add an example file to show how you generate badges, but I don't want to include express as a dependency of this project.  This can be solved via documentation, and usage of npx.